Nodejs/Product/TestAdapter/TestDiscovererWorker.cs

```diff
@@ -26,14 +38,20 @@ public TestDiscovererWorker(string testSource, string nodeExePath)
             this.nodeExePath = nodeExePath;
         }
 
-        public void DiscoverTests(string testFolderPath, TestFramework testFx, IMessageLogger logger, ITestCaseDiscoverySink discoverySink)
+        public void DiscoverTests(string testFolderPath, TestFramework testFx, IMessageLogger logger, ITestCaseDiscoverySink discoverySink, string testDiscovererName)
         {
             var fileList = Directory.EnumerateFiles(testFolderPath, "*.js", SearchOption.AllDirectories).Where(x => !x.Contains(NodejsConstants.NodeModulesFolder));
-            this.DiscoverTests(fileList, testFx, logger, discoverySink);
+            this.DiscoverTests(fileList, testFx, logger, discoverySink, testDiscovererName);
         }
 
-        public void DiscoverTests(IEnumerable<string> fileList, TestFramework testFx, IMessageLogger logger, ITestCaseDiscoverySink discoverySink)
+        public void DiscoverTests(IEnumerable<string> fileList, TestFramework testFx, IMessageLogger logger, ITestCaseDiscoverySink discoverySink, string testDiscovererName)
         {
+            // If it's a framework name we support, is safe to submit the string to telemetry.
+            if (supportedFrameworks.Contains(testFx.Name))
```

Comment:
It might be interesting to report "other" if it's not a supported framework, so we get a sense of how common that is.

Reply:
Good idea. Added,